packages/api/tsconfig.json

```diff
@@ -12,7 +12,7 @@
     "allowUnreachableCode": false,
     "noFallthroughCasesInSwitch": true,
     "noImplicitOverride": true,
-    "noUnusedLocals": true,
+    "noUnusedLocals": false,
```

Comment:
Why are we setting this to false?

Reply:
good catch! this should not be updated, reverting